lib/chef/chef_fs/file_system.rb

```diff
@@ -292,7 +292,7 @@ def self.copy_entries(src_entry, dest_entry, new_dest_parent, recurse_depth, opt
                   end
                 end
               else
-                ui.output ("Not deleting extra entry #{dest_path} (purge is off)") if ui
+                ui.output "Not deleting extra entry #{dest_path} (purge is off)" if ui
```

Comment:
In my experience, we prefer parentheses around method arguments, but without the leading space. So this should be:
```
ui.output("Not deleting extra entry #{dest_path} (purge is off)") if ui
``` 

IMHO, it certainly reads better when you've got a conditional on the end because you're less tempted to write that conditional off as part of the string. That said @jkeiser wrote most of this file and didn't use parentheses. In any case, we're just bikeshedding on style.